### ISSUE TITLE
[EN][Inscription] Book of glyph mastery

### ISSUE THREAD

**Issue by ramalo** [Atlantiss/BugTracker#4502]
Hi everyone,

First of all, I wanna excuse myself because my english cant be the best one. I dont really know if there is another post about this topic but adter searching it I think there isnt. I'm gonna talk about this great inscription books that can be looted from Northrend mobs. Most people know that the best place to farm these books are "Converted Hero"s and "Scourge Banner-Bearer" from the west part of Icecrown, or at least most people think it.

Here I wanna show you the drop rates of each mob: 
http://www.wowhead.com/npc=31900/scourge-banner-bearer    http://www.wowhead.com/npc=32255/converted-hero
As you can see, the percentage is just a 0,05 and a 0,07. Well, I may have gone there maybe 9-10 times to full my inventory (4x16 stacks) and I have got no book. However I have got 10 Damaged Necklaces, which drop rate is just a 0,02%.

I feel sad because I have no screenshoot to show that it is true but I think it is awesome that an item with a 0,02% has more chances to be dropt than one with 0,05% or 0,07%. I think something might be happening with the drop rates of the book, maybe they are smaller than they should be. Anyways, I think someone should increase these rates because it is almost impossible to get the 58 books that an inscripter needs to learn all the glyphs from these books (of course if u are rich and u can pay 58*5000=290,000g to get all the books there is no problem). I dont know, but I think no other profession has this type of problem and I suggest to make this easier.

I hope someone will fix this/make this easier for inscription.

**Comment by sigmundfreud2016** [comment]
There must be another place to farm them cuz I saw one man selling like 10-15 of them on AH. It would take a lifetime to get so much books from usual mobs. But the problem exist. Broken chests make it even worse. I could get a book every day simply by looting these guys http://www.wowhead.com/object=207517/silken-treasure-chest http://www.wowhead.com/object=207493/sturdy-treasure-chest but they don't respawn at all.

**Comment by Crusaderz** [comment]
This is really big problem. Each book cost ~8k on AH. There is 57 glyphs unique to that book. It means every scribe should spend 456k in order to get all recipes. No other profession requires such expenses.

And on retail during cataclysm, books cost 500-1k (depending on server). So why prices here much more higher? There is a number of reasons.
1) The real drop chance for them are unknown. And current drop chances are not blizzlike, they much lower. The drop in WOTLK instances wasn't uncommon on retail, while on atlantiss they uber-rare. I spent 68-80 levels mostly in instances and seen it only once (in nexus).

comment from wowhead (patch 4.3.4):
I've been farming this out of Pit of Saron, the icc lvl 80 dungeon. With my Blood DK I can clear this instance in about 12-15 minutes in normal mode. I've been getting 1 of these books every 4 runs, in a couple hours I had 4 already, they drop of any mob in the place, so make sure you kill everything. So, if there's a way to "farm" these things, that dungeon is your best choice.

4.1.0
Seems to have a ~1% droprate from Converted Heroes in Icecrown, I've been farming them for cloth for some time and gotten 12 books from about 1300 kills.Not much, but probably the best spot if you want/need to farm these.

2) The treasure chests, another source of books, not working and not spawning on Atlantiss.
The chests have good chance to drop books http://www.wowhead.com/item=45912/book-of-glyph-mastery#contained-in-object and according to number of wowhead comments (4.3.4) chests are very reliable source of books:

I just tried this and it worked very well for me, i just got 2 books in less then an hour using this method. ( do note that im on a very low populated server so all the chests were pretty much mine. )
--------------------------------------------------------------------
I usually do 1-2 treasure runs a day and find maybe 5-7 treasure chests in one day. The zones I go thru are Sholazar Basin, Icecrown, Crystal Song and Stormpeaks. From my own experience, I'd say its about 1 in 7-8 I would get a Book of Glyph Mastery. Some days I get two books. Some days I get none. To fly thru each zone it probably takes 10 minutes per zone.
--------------------------------------------------------------------
These drop quite often from treasure chests located in high level Northrend zones such as Icecrown, Storm Peaks and Sholazar Basin. I usually check all the spawn locations marked by GatherMate2 addon daily and usually get 1-3 per run.
Hope this helps anyone trying to farm these.



In short: Increase drop chances and fix chests + add books to their loot table. Or make ALL glyphs discovered through research. Blizzard promised to do it in patch 4.2 (http://us.battle.net/wow/en/forum/topic/2522165843), but delayed it until Mists of Pandaria. 

Sorry for my terrible english.

**Comment by mikonek07** [comment]
Still waiting for implementing fix.
